package.json

```diff
@@ -11,7 +11,7 @@
     "test": "buidler test",
     "compile": "buidler --show-stack-traces --max-memory 8192 compile",
     "start": "buidler node --port $LOCAL_BUIDLEREVM_PORT",
-    "coverage": "rm -rf build cache test-results.xml && buidler compile && buidler coverage --network coverage --temp build; rm -rf cache",
+    "coverage": "rm -rf build cache test-results.xml && buidler compile && buidler coverage --network coverage --temp build",
```

Comment:
keep the `; rm -rf cache` as the coverage doesn't seem to clean up after itself

Reply:
Fixed, but I didn't remove this.